imports/api/settings.js

```diff
@@ -74,6 +75,10 @@ Meteor.methods({
     if (this.userId) {
       let user = Meteor.users.findOne({_id: this.userId})
       if (user.hasGreaterRole(ROLES.admin)) {
+        if ((settings.email !== Settings.findOne().email) && process.env.QLICKER_MAIL_PASSWORD) {
```

Comment:
Also, I don't think you want to set process.env.MAIL_URL; that should get passed in as an environment variable. The only one we want to set in the settings is the "from", which was previously hardcoded to enoch's server